.github/workflows/rust.yml

```diff
@@ -148,6 +148,29 @@ jobs:
         run: |
           ./android-build cargo build --target="armv7-linux-androideabi"
 
+  ohos:
+    name: "OpenHarmony build"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [ 'aarch64-unknown-linux-ohos' ]
```

Comment:
nit-pick: rename field to target (arch is only part of target)

Reply:
done